builtin/solidity/context.go

```diff
@@ -34,3 +34,7 @@ func (c *Context) UseGas(gas uint64) {
 		c.charger.Charge(gas)
 	}
 }
+
+func (c *Context) Address() thor.Address {
+	return c.address
+}
```

Comment:
Probably ok to store the address or the `sctx := solidity.NewContext(addr, state, charger)` in a property of the Staker Object